CMakeLists.txt

```diff
@@ -218,10 +218,18 @@ if (PROJECT_IS_TOP_LEVEL)
   install(TARGETS ${NVFUSER_CODEGEN} EXPORT NvfuserTargets DESTINATION lib)
 
   # We are keeping fusion_cache_generated.h for the submodule build because flatc is unavailable.
-  add_custom_target(build_flatbuffer_config
-      COMMAND ./third_party/flatbuffers/flatc -o ../csrc/serde/ -c -b ../csrc/serde/fusion_cache.fbs
-      DEPENDS flatc
-      COMMENT "Generating fusion_cache_generated header from fusion_cache.fbs")
+  add_custom_command(
+    OUTPUT
+      ${NVFUSER_ROOT}/csrc/serde/fusion_cache_generated.h
+    DEPENDS
+      ${NVFUSER_ROOT}/csrc/serde/fusion_cache.fbs
+    DEPENDS flatc
+    COMMAND ./third_party/flatbuffers/flatc -o ../csrc/serde/ -c -b ../csrc/serde/fusion_cache.fbs
```

Comment:
Should we use `${NVFUSER_ROOT}` here?

Reply:
Not sure what the best way is to refer to that path. @jjsjann123 do you have any thoughts?

Reply:
Oh I see what you mean. Just for the path arguments. Pushed a change that does that and refers to flatc binary location inside `${CMAKE_CURRENT_BINARY_DIR}`.